test/linalg/symmetric.jl

```diff
@@ -72,10 +72,6 @@ end
                 @test isa(similar(Symmetric(asym), Int, (3,2)), Matrix{Int})
                 @test isa(similar(Hermitian(aherm), Int, (3,2)), Matrix{Int})
             end
-            @testset "full" begin
```

Comment:
Worth keeping in some form? Unleds `Matrix(S::Symmetric)` is tested elsewhere in this file

Reply:
Didn't find this explicitly tested so took the liberty to add it.

Reply:
Good call, and thanks for the lovely addition! :)